javatests/arcs/core/data/TypeUtilsTest.kt

```diff
@@ -213,21 +361,47 @@ class TypeUtilsTest {
       FieldType.Text to InferredType.Primitive.TextType
     )
 
-    // This test schema is used for the Type.toSchema tests below. Its contents are not relevant
+    // This test schema is used for the Type.toSchema tests above. Its contents are not relevant
     // to the tests, the goal was to populate the fields with some "typical" values.
     private val DUMMY_SCHEMA = Schema(
-      names = setOf(SchemaName("name1"), SchemaName("name2")),
+      names = setOf(SchemaName("Person"), SchemaName("NickNames")),
       fields = SchemaFields(
         singletons = mapOf(
-          "name" to FieldType.Text
+          "name" to FieldType.Text,
+          "age" to FieldType.Int,
+          "birthYear" to FieldType.Int
         ),
         collections = mapOf(
-          "item" to FieldType.EntityRef("abc")
+          "nick" to FieldType.EntityRef("abc")
         )
       ),
       hash = "abc123",
-      refinementExpression = false.asExpr(),
-      queryExpression = false.asExpr()
+      refinementExpression = true.asExpr(),
+      queryExpression = true.asExpr()
+    )
+    private val REFERENCED_DUMMY_SCHEMA = Schema(
+      names = setOf(SchemaName("NickName")),
+      fields = SchemaFields(
+        singletons = mapOf(
+          "name" to FieldType.Text
+        ),
+        collections = emptyMap()
+      ),
+      hash = "abc",
+      refinementExpression = true.asExpr(),
+      queryExpression = true.asExpr()
+    )
+    private val RESULT_SCHEMA = Schema(
+      names = setOf(SchemaName("Test")),
+      fields = SchemaFields(
+              singletons = mapOf(
+                      "result" to FieldType.Int
+              ),
+              collections = emptyMap()
```

Comment:
This indentation looks off. Can you please run the formatter over the files you've changed?

Reply:
It seems my .aswb file had a indent=4, changed to 2 and reformatted.